src/Common/src/TypeSystem/Common/MetadataFieldLayoutAlgorithm.cs

```diff
@@ -158,11 +160,12 @@ out instanceByteSizeAndAlignment
 
             // At this point all special cases are handled and all inputs validated
 
-            if (type.IsExplicitLayout)
+            if (!TargetCoreCLR && type.IsExplicitLayout)
```

Comment:
Have we ever talked about having an Abi value in TargetAbi enum?

Reply:
I think we had it at some point but then it became unnecessary after Simon created the split CorInfoImpl.ReadyToRun / RyuJit pair. Perhaps we might be able to expose this flag in the forked files and query it from here.

Reply:
Or we could add the new TargetAbi value, I agree that would be probably more systematic.

Reply:
TargetCoreCLR removed in 2nd commit.